src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md

```diff
@@ -22,9 +22,9 @@ Set-AzVmssDiskEncryptionExtension [-ResourceGroupName] <String> [-VMScaleSetName
 ```
 
 ## DESCRIPTION
-The **Set-AzVmssDiskEncryptionExtension** cmdlet enables encryption on a VM scale set.
-This cmdlet enables encryption by installing the disk encryption extension on the VM scale set.
-If no *Name* parameter is specified, an extension with the default name AzureDiskEncryption for virtual machines that run the Windows operating system or AzureDiskEncryptionForLinux for Linux virtual machines are installed.
+The **Set-AzVmssDiskEncryptionExtension** cmdlet enables encryption on a VM scale set. This cmdlet enables encryption by installing the disk encryption extension on the VM scale set.
+
+The *VolumeType* parameter is required when encrypting Linux virtual machines, and must be set to "Data". It can be omitted when encrypting Windows virtual machines.
```

Comment:
must be set to "Data" for VMSS.   

Even if the single VM version of ADE would support OS disk encryption, VMSS currently only supports data disk encryption.

Reply:
Since these examples are separate between VMSS and VM, I think it maybe fine as written.

Reply:
Yes, this is the AzVMSS cmdlet; revisions to the AzVM cmdlets are also in the PR